web/components/FcDrawerViewData.vue

```diff
@@ -331,6 +315,9 @@ export default {
       this.studySummary = studySummary;
       this.loadingStudies = false;
     },
+    locationMode() {
+      this.detailView = false;
```

Comment:
we need to make sure this is consistently using `detailMode` or `detailView`, not both.

(my vote: `detailView`, since that avoids confusion with `locationMode`)